db/seeds.rb

```diff
@@ -30,26 +30,23 @@
   end
 end
 
-def create_mail_method
-  Spree::MailMethod.destroy_all
-
-  CreateMailMethod.new(
-    environment: Rails.env,
-    preferred_enable_mail_delivery: true,
-    preferred_mail_host: ENV.fetch('MAIL_HOST'),
-    preferred_mail_domain: ENV.fetch('MAIL_DOMAIN'),
-    preferred_mail_port: ENV.fetch('MAIL_PORT'),
-    preferred_mail_auth_type: 'login',
-    preferred_smtp_username: ENV.fetch('SMTP_USERNAME'),
-    preferred_smtp_password: ENV.fetch('SMTP_PASSWORD'),
-    preferred_secure_connection_type: 'None',
-    preferred_mails_from: ENV.fetch('MAILS_FROM', "no-reply@#{ENV.fetch('MAIL_DOMAIN')}"),
-    preferred_mail_bcc: ENV.fetch('MAIL_BCC', ''),
-    preferred_intercept_email: ''
-  ).call
+def set_mail_configuration
+  MailConfiguration.entries=({
+    enable_mail_delivery: true,
+    mail_host: ENV.fetch('MAIL_HOST'),
+    mail_domain: ENV.fetch('MAIL_DOMAIN'),
+    mail_port: ENV.fetch('MAIL_PORT'),
+    mail_auth_type: 'login',
+    smtp_username: ENV.fetch('SMTP_USERNAME'),
+    smtp_password: ENV.fetch('SMTP_PASSWORD'),
+    secure_connection_type: 'None',
+    mails_from: ENV.fetch('MAILS_FROM', "no-reply@#{ENV.fetch('MAIL_DOMAIN')}"),
+    mail_bcc: ENV.fetch('MAIL_BCC', ''),
+    intercept_email: ''
+  })
```

Comment:
I tried that (because the previous createmethod.new call didnt have them), but it gives me this error on rake db:seeds:
SyntaxError: db/seeds.rb:35: syntax error, unexpected tLABEL
    enable_mail_delivery: true,

I dont understand why the general rule of omitting the curly braces doesnt work here...